src/EventTickets/resources/admin/components/EventTicketsListTable/EventTicketsRowActions.tsx

```diff
@@ -32,13 +34,15 @@ export function EventTicketsRowActions({item, setUpdateErrors, parameters}) {
                 href={`edit.php?post_type=give_forms&page=give-event-tickets&id=${item.id}`}
                 displayText={__('Edit', 'give')}
             />
-            <RowAction
-                onClick={confirmModal}
-                actionId={item.id}
-                displayText={__('Delete', 'give')}
-                hiddenText={item.title}
-                highlight
-            />
+            {salesCount === 0 && (
+                <RowAction
+                    onClick={confirmModal}
+                    actionId={item.id}
+                    displayText={__('Delete', 'give')}
```

Comment:
@jdghinson what are your thoughts on the UX of removing the "delete" button when an event cannot be deleted?

My concern is that it could be confusing if there is no explanation.

Reply:
Reverted that and implemented a native alert for now (agreed with Jeffrey). https://github.com/impress-org/givewp/pull/7292/commits/9bb51df75b653b3d5d476850a059afa2c4918868